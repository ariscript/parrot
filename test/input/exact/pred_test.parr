def less_than(x, y): x < y

check:
    3 is%(less_than) 4,
    1 is%(less_than) 3 because 2.

nil

