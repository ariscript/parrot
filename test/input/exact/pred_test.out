check block: 2/2 tests passed
nil
